README.md

```diff
@@ -15,6 +15,24 @@ $ cd storageos
 $ helm install .
 ```
 
+IMPORTANT:  The StorageOS api address must be manually set in the `api.address`
+value.  Set this to the ip address of one of the Kubernetes nodes, e.g. `http://10.0.0.1`.
```

Comment:
http://10.0.0.1:**5705**

Reply:
done, though the port is optional when it hasn't been changed